src/RentableCar.java

```diff
@@ -0,0 +1,45 @@
+public class RentableCar extends Car implements Rentable {
+    private Person person;
+
+    public RentableCar(String name, int year, int seatsNumber) {
+        super(name, year, seatsNumber);
+        this.person = new Person(null, null, null);
+    }
+
+    public Person getPerson() {
+        return person;
+    }
+
+    public void setPerson(Person person) {
+        this.person = person;
+    }
+
+    @Override
+    public void rent(String firstName, String lastName, String id) {
+        this.person = new Person(firstName, lastName, id);
+
+    }
+
+    @Override
+    public void handOver() {
+        this.person.setFirstName(null);
```

Comment:
tu powinienies ustawic cale pole person na `null` . To lepiej reprezentuje fakt ze zadna osoba nie wypozycza wlansie samochodu

Reply:
OK

Reply:
OK